confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeDescriptor.java

```diff
@@ -42,42 +43,52 @@ public ConfigNodeConf getConf() {
     return conf;
   }
 
-  public String getPropsDir() {
```

Comment:
Why delete this method? For distinguish ConfigNode's environment variables from DataNode's, I set CONFIGNODE_NODE_CONF and CONFIGNODE_NODE_HOME in the start-confignode scripts. Now we can see that the scripts output "WARN org.apache.iotdb.confignode.conf.ConfigNodeDescriptor - Cannot find IOTDB_HOME or IOTDB_CONF environment variable when loading config file iotdb-confignode.properties, use default configuration". This is a bug needs to be fixed.

Reply:
fixed